test/browser_tests/page_objects/base-page.js

```diff
@@ -1,29 +1,65 @@
 class BasePage {
-  gotoURL( url ) {
+  async gotoURL ( url ) {
     const gotoUrl = url || this.URL || '/';
+    await browser.get( gotoUrl );
+    await this.setElements();
+    await this.disableAnimations();
+    await this.removeExternalScripts();
 
-    return browser.get( gotoUrl )
-      .then( function() {
-        BasePage.dismissAlert( gotoUrl );
-      } );
+    return BasePage.dismissAlert( gotoUrl );
   }
 
-  static dismissAlert() {
+  removeExternalScripts() {
+
+    /**
+     * Remove any exernal scripts.
+     * @param {Object} browser Protractor browser object.
+     */
+    function _removeScripts() {
+      [].slice.call( document.querySelectorAll( 'script, style, link' ) )
+      .filter( script => {
+        const src = script.href || script.src;
+
+        return src && src.indexOf( 'localhost' ) === -1;
+      } )
+      .forEach( script => script.parentNode.removeChild( script ) );
+    }
+
+    return browser.executeScript( _removeScripts );
+  }
+
+  disableAnimations() {
+
+    /**
+     * Disable CSS3 animations in when running tests.
+     * @param {Object} browser Protractor browser object.
+     */
+    function _disableAnimations() {
+      const style = document.createElement( 'style' );
+      style.type = 'text/css';
+      style.innerHTML = '* { transition-duration: .1ms !important; }';
+      document.body.appendChild( style );
+    }
+
+    return browser.executeScript( _disableAnimations );
+  }
+
+  setElements() { } // eslint-disable-line no-empty-function
```

Comment:
Let's put a code comment in the function that describes why it is empty and remove the eslint disabled rule.

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3946/commits/7c6049fbf5872dd105ede2ba05512b407babad55.